pkg/vadmin/vc_errors.go

```diff
@@ -65,8 +78,16 @@ func (v *vcErrors) LogFailure(cmd string, err error) (ctrl.Result, error) {
 	}
 	v.EVWriter.Eventf(v.VDB, corev1.EventTypeWarning, reason, vproblem.Title)
 	if !ok {
-		return ctrl.Result{}, fmt.Errorf("failed command %s: %w", cmd, err)
+		return ctrl.Result{}, fmt.Errorf("failed command %s: %w", cmd, vproblem)
 	}
 	// All known errors we return with requeue set to true.
 	return ctrl.Result{Requeue: true}, nil
 }
+
+func (v *vcErrors) logClusterLeaseNotExpiredError(err *vclusterops.ClusterLeaseNotExpiredError) (ctrl.Result, error) {
+	v.Log.Info("vclusterOps command failed because the cluster lease was not expired", "msg", err.Error())
+	v.EVWriter.Eventf(v.VDB, corev1.EventTypeWarning, events.ReviveDBClusterInUse,
```

Comment:
What about `ReviveDBNodeCountMismatch`, `ReviveDBPermissionDenied`, `ReviveDBNotFound`, are we going to get them from server through Rfc7807?

Reply:
It depends on where those errors originate. It's likely the last two originate from the https endpoint, so they will be rfc7807 errors. Not sure where the first one is detected.

Reply:
Some more details where these errors originate. The first one is detected in vclusterops, so we can return a specific error struct for that. I'll log this in the jira's we have for this work so we don't lose track.

ReviveDBNodeCountMismatch
[NMADownloadFileOp] nodes mismatch found on host 10.244.0.184: the number of the new nodes in --hosts is 4, but the number of the old nodes in description file is 3\ncould not find a host with a passing result"

ReviveDBPermissionDenied
[NMADownloadFileOp] HTTPS call failed on host 10.244.0.222
status code 500 returned from host 10.244.0.222: Error happened during executing vertica-download-file. Stdout: . Stderr:Initialization thread logged exception:
Could not copy file [s3://spilly/here/metadata/vertdb/cluster_config.json] to [/tmp/desc.json]: curlCode: 28, Timeout was reached                                                             Exiting process ( exit(1) ).. Return code: 1. Complete error info: exit status 1 

ReviveDBNotFound
[NMADownloadFileOp] HTTPS call failed on host 10.244.0.196\nstatus code 500 returned from host 10.244.0.196: Error happened during executing vertica-download-file. Stdout: . Stderr:Initialization thread logged exception:\nCould not copy file [/communal/0ae3e837-3167-402f-a8f7-316ffdfd65f3z/metadata/vertdb/cluster_config.json] to [/tmp/desc.json]: No such file or directory\nExiting process ( exit(1) ).. Return code: 1. Complete error info: exit status 1\n\ncould not find a host with a passing result"

Reply:
The last 2 are from the nma right? not the embedded https server.

Reply:
The nma just copies out the result from the call to vertica-donwload-file. So, ultimately somewhere in the server this error is generated.